geomesa-features/geomesa-feature-common/src/main/scala/org/locationtech/geomesa/features/serialization/WkbSerialization.scala

```diff
@@ -27,8 +28,13 @@ trait WkbSerialization[T <: NumericWriter, V <: NumericReader] {
 
   import WkbSerialization._
 
-  private lazy val factory = new GeometryFactory()
-  private lazy val csFactory = factory.getCoordinateSequenceFactory
+  private val factory = new GeometryFactory()
+  private val csFactory = CoordinateArraySequenceFactory.instance()
+
+  private val xySerializer   = new XYSerializer()
+  private val xyzSerializer  = new XYZSerializer()
+  private val xymSerializer  = new XYMSerializer()
+  private val xyzmSerializer = new XYZMSerializer()
```

Comment:
Where all does this trait get mixed in?  Should these be lazy vals?  Do we care?

Reply:
it's mixed into an object, so i figured it was better to not have them be lazy